waggle-dance-integration-tests/src/test/resources/log4j2.xml

```diff
@@ -17,7 +17,7 @@
 <Configuration>
   <Appenders>
     <Console name="STDOUT" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{ISO8601} %tn %-5p %c:%L - %m%n"/>
+      <PatternLayout pattern="%d{ISO8601} %t %-5p %c:%L - %m%n"/>
```

Comment:
What is the change here?

Reply:
It printed an "n" after the thread name, I think this was just a typo :)